src/components/Button.tsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react/prop-types */
+import * as React from "react";
+import { Link } from "react-router-dom";
+
+interface Props {
+  children: unknown;
+  className: string;
+  linkInfo: { isRouterLink: boolean; isLink: boolean };
+  to?: string;
+}
+
+const Button: React.FC<Props> = ({
+  children,
+  className,
+  linkInfo: { isRouterLink, isLink },
+  to,
+}): JSX.Element => {
+  if (isLink)
+    return (
+      <a href={to} className={`button ${className}`}>
+        {children}
+      </a>
+    );
+
+  if (isRouterLink)
+    return (
+      <Link to={to as unknown as never} className={`button ${className}`}>
```

Comment:
Codacy has a fix for the issue: [Replace `to·as·unknown` with `(to·as·unknown)`](https://app.codacy.com/gh/xeptao/xeptao.com/pullRequest?prid=7550898)
```suggestion
      <Link to={(to as unknown) as never} className={`button ${className}`}>
```